docs/src/main/sphinx/installation/containers.rst

```diff
@@ -1,9 +1,11 @@
-===================
-Trino in containers
-===================
+===========================
+Trino in a Docker container
+===========================
 
 The Trino project provides the `trinodb/trino <https://hub.docker.com/r/trinodb/trino>`_
-Docker image that includes the Trino server and a default configuration.
+Docker image that includes the Trino server and a default configuration and
+publishes it to Docker Hub. The image can be used in the Docker runtime, among
```

Comment:
Docker image includes ... and publishes ... ?

Reply:
Its confusing because the sentence is long and misses some commas. Essentially .. its the Trino project provided and published.

How about


The Trino project provides the trinodb/trino Docker image that includes the Trino server and a default configuration. The Docker image is published to Docker Hub and can be used with the Docker runtime, among ...